learn/core_concepts/documents.md

```diff
@@ -169,6 +169,18 @@ There are several ways to set the primary key for an index:
 - You can set it manually [on document addition](/reference/api/documents.md#add-or-replace-documents)
 - If no primary key is set, MeiliSearch automatically [guesses the primary key](/learn/core_concepts/documents.md#meilisearch-guesses-your-primary-key) when you add documents
 
+#### Setting the primary key on index creation
+
+The code below creates an index called `movies` with `id` as primary key:
```

Comment:
```suggestion
The code below creates an index called `movies` with `id` as primary key:
```

Maybe to follow up on the reason why you use the `primaryKey` field it would be better to use `reference_number` as the primaryKey in this example as well?